python.html.markdown

```diff
@@ -734,6 +735,26 @@ print(gen_to_list)  # => [-1, -2, -3, -4, -5]
 
 
 # Decorators
+# Decorator is a higher order function, which accepts and returns a function.
+# In our case decorator accepts a function that is wrapped by decorator as a
```

Comment:
I don't understand this sentence – is the first "decorator" supposed to be "add_apples"? Explaining what is a decorator with "decorator" is confusing :)